tests/test_login_to_rps.py

```diff
@@ -98,3 +114,32 @@ def test_login_servicenow_stage(self, selenium, ldap, urls, counter_api):
                                                                        conftest.increase_otp_counter(counter_api)))
         assert test.is_profile_icon_displayed
         test.logout()
+
+    def test_login_biztera(self, selenium, ldap, urls, counter_api):
```

Comment:
All three following methods are doing the same thing. Please deduplicate the code and override appropriately.